versioned_docs/version-1.1.0/quick_start.md

```diff
@@ -0,0 +1,101 @@
+---
+id: quick_start
+title: Quick Start
+---
+
+To start a new task locally, in development mode follow these steps:
+
+
+### 1. Start your new task from our template repository
+
+The simplest way to get started is creating a new repository using Honeycomb as a template.
+
+Go to https://github.com/brown-ccv/honeycomb and click on `Use this template` on the top right. Then select the organization and the name of your repository and click on `create repository from template`
+
+Alternatively, you can use GitHub CLI to create a new project based on the Honeycomb template repository. First, install GitHub CLI (https://cli.github.com/), then simply run on your terminal: 
+
+```
+gh repo create your-new-task-name --template brown-ccv/honeycomb
+``` 
+
+You can now move into the directory that was just created
+
+```
+cd your-new-task-name
+```
+
+### 2. Change name and description
+
+Update the `package.json` fields to reflect your app name and description, e.g. `name`, `author`, `repository`
+
+### 3. Install the dependencies. 
+
+**Electron Dependencies**
+Honeycomb relies on Electron to package the cross-platform desktop applications. Before starting, you will need to install Electron's pre-requisites. You can find instructions [here](https://www.electronjs.org/docs/development) for your specific OS
```

Comment:
Do you mind adding the requirements more explicitly? See [this](https://github.com/brown-ccv/honeycomb-docs/commit/2b7bcc44bc21b32d8d16fa7b2d63e8034f71ff86) for reference.
Still point o the electron link as a reference to up-to-date instructions. Thanks!